Content.Server/Remotes/DoorRemoteSystem.cs

```diff
@@ -36,13 +36,17 @@ public void OnInHandActivation(EntityUid user, DoorRemoteComponent component, Us
             switch (component.Mode)
             {
                 case OperatingMode.OpenClose:
-                    component.Mode = OperatingMode.ToggleBolts;
-                    switchMessageId = "door-remote-switch-state-toggle-bolts";
-                    break;
+                        component.Mode = OperatingMode.ToggleBolts;
+                        switchMessageId = "door-remote-switch-state-toggle-bolts";
+                        break;
+
+                    // Skip toggle bolts mode and move on from there (to emergency access)
                 case OperatingMode.ToggleBolts:
-                    component.Mode = OperatingMode.ToggleEmergencyAccess;
-                    switchMessageId = "door-remote-switch-state-toggle-emergency-access";
-                    break;
+                        component.Mode = OperatingMode.ToggleEmergencyAccess;
+                        switchMessageId = "door-remote-switch-state-toggle-emergency-access";
+                        break;
+
+                    // Skip ToggleEmergencyAccess mode and move on from there (to door toggle)
```

Comment:
this is double indented

Reply:
Result of merging and whatnot, good catch thanks.